### PR TITLE
Update openbr.rb

### PR DESCRIPTION
**Pull request by shashanksingh**
Change version from 0.4.1 to 0.5 with appropriate download url.

### CODE REVIEW

File: openbr.rb [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 class Openbr < Formula
   homepage "http://www.openbiometrics.org/"
-  url "https://github.com/biometrics/openbr/releases/download/v0.4.1/openbr0.4.1_withModels.tar.gz"
-  version "0.4.1"
-  sha1 "1cf3b9b6fad1c717d8b0256b3a22d138510572e9"
+  url "https://github.com/biometrics/openbr/releases/download/v0.5.0/OpenBR-0.5.0-osx.zip"
+  version "0.5.0"
```
Comment (iml) [inline openbr.rb]: Please update the checksum. Also, we need the source archive not precompiled binaries.

Comment (shashanksingh) [inline openbr.rb]: Sure , thanks will do.

Comment (dpo) [inline openbr.rb]: Please also use the `sha256` instead of `sha1`.

**Comment by dpo** [comment]
@shashanksingh

**Pull request closed without merging**